cafebabel/templates/articles/tags/detail.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+
+{% block meta_title %}{{ tag.name }}{% endblock %}
+{% block meta_description %}{{ tag.summary }}{% endblock %}
```

Comment:
These probably need to be enriched with the extra metas (ie: the image)

Reply:
@cafebabel-webmaster can you take care of this?

Reply:
ok done.